packages/ui/src/Checkbox.module.scss

```diff
@@ -23,8 +23,8 @@
 
   // hide native checkbox
   input {
-    height: 0;
-    width: 0;
+    height: 1px;
```

Comment:
I've learned that some older browsers had problems with reading this if height/width is 0px. It might not be a problem today, but it should be safer.